server/src/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -1228,12 +1230,22 @@ public UserVm addNicToVirtualMachine(AddNicToVMCmd cmd) throws InvalidParameterV
         return _vmDao.findById(vmInstance.getId());
     }
 
+    /**
+     * It checks if the given mac is null, in case of a null mac it calls method  {@link com.cloud.network.NetworkModel #getNextAvailableMacAddressInNetwork(long)} and returns the generated mac address 
```

Comment:
"MAC" is an acronym. Therefore, would you mind capitalizing it in the Java documentation?